app/src/main/java/com/gophillygo/app/data/models/AttractionFlag.java

```diff
@@ -0,0 +1,92 @@
+package com.gophillygo.app.data.models;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+import android.arch.persistence.room.TypeConverter;
+import android.arch.persistence.room.TypeConverters;
+import android.support.annotation.DrawableRes;
+import android.util.SparseArray;
+
+import com.google.gson.annotations.SerializedName;
+import com.gophillygo.app.R;
+
+@Entity
+public class AttractionFlag {
+
+    public enum Option {
+        NotSelected (0, R.drawable.ic_add_black_24dp),
```

Comment:
This mapping is for the black icon only. There are also white versions of these icons, and we may add versions on other colors (see wireframes, like [here](https://marvelapp.com/4c295bb/screen/37237000)). At least if we change the color of the current selection in the popover, we'll need the white icon resource.